controlpanel/api/models.py

```diff
@@ -24,6 +22,7 @@ class User(AbstractUser):
     REQUIRED_FIELDS = ['email', 'auth0_id']
 
     class Meta:
+        db_table = 'control_panel_api_user'
```

Comment:
The models all specify the old table names so that a bunch of duplicate tables called `api_user`, etc are not created.

### [ ⏩ Next ]()